src/core/classes/View.php

```diff
@@ -115,7 +115,7 @@ static function script($script, $uri=false, $prop='')
   {
     if(in_array($script, self::$script)) return;
     self::$script[]=$script;
-    if(Gila::config('use_cdn')=='1' && isset(self::$cdn_paths[$script])) {
+    if(@isset(self::$cdn_paths[$script])) {
```

Comment:
error suppressing isn't needed so the line could be written `if(isset(self::$cdn_paths[$script])) {`